generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/BUILD

```diff
@@ -3,20 +3,17 @@ java_library(
     srcs = glob(["*.java"]),
     visibility = ["//generate_workspace/src:__subpackages__"],
     runtime_deps = [
-        "@org_codehaus_plexus_plexus_interpolation//jar",
-        "@org_codehaus_plexus_plexus_utils//jar",
+        "//third_party:org_codehaus_plexus_plexus_interpolation",
```

Comment:
No, we can't remove those lines from the WORKSPACE file.  They are what actually import the Maven jars!  The //third_party targets just depend on the maven_jars.  

I changed it to use the libraries because they are user-friendlier: suppose there is a Maven artifact a:a:1.0 that depends on b:b:1.0.  If you're depending on @a_a//jar, your java_library has to also depend on @b_b//jar or you could get an error when your code calls something in @a_a//jar and @b_b//jar isn't in the classpath.  

The java_library definitions are generated from a:a:1.0's list of dependencies, so depending on //third_party:a_a gives you the dependency on @b_b//jar, too.